packages/docs/docs/tutorial/integrate-sdk/react.mdx

```diff
@@ -133,7 +135,7 @@ const Component = () => {
 
 ## Sign Out
 
-Calling `.signOut()` will clean all the Logto data in memory and LocalStorage, if it has.
+Calling `.signOut()` will clear all the Logto data in memory and localStorage, if there is any.
```

Comment:
nit: maybe better `if there is any` → `if they exist`?

Reply:
hmmm, `if there is any` does sound a bit informal. I'll take your suggestion, thanks